src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -1422,32 +1442,36 @@ void ReportCoilSelection::setCoilHeatingCapacity(
             c->coilDesEntEnth = Psychrometrics::PsyHFnTdbW(c->coilDesEntTemp, c->coilDesEntHumRat);
             c->coilDesLvgTemp = DataSizing::FinalSysSizing(curSysNum).PreheatTemp;
             c->coilDesLvgHumRat = DataSizing::FinalSysSizing(curSysNum).PreheatHumRat;
-            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress,
-                                                                  "ReportCoilSelection::setCoilCoolingCapacity");
+            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
+                c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
             c->coilDesLvgEnth = Psychrometrics::PsyHFnTdbW(c->coilDesLvgTemp, c->coilDesLvgHumRat);
 
         } else { // part of main air loop
             c->coilDesEntTemp = DataSizing::FinalSysSizing(curSysNum).HeatMixTemp;
             c->coilDesEntHumRat = DataSizing::FinalSysSizing(curSysNum).HeatMixHumRat;
-            c->coilDesEntWetBulb = Psychrometrics::PsyTwbFnTdbWPb(c->coilDesEntTemp, c->coilDesEntHumRat, DataEnvironment::StdBaroPress,
-                                                                  "ReportCoilSelection::setCoilCoolingCapacity");
+            c->coilDesEntWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
+                c->coilDesEntTemp, c->coilDesEntHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
             c->coilDesEntEnth = Psychrometrics::PsyHFnTdbW(c->coilDesEntTemp, c->coilDesEntHumRat);
             c->coilDesLvgTemp = DataSizing::FinalSysSizing(curSysNum).HeatSupTemp;
             c->coilDesLvgHumRat = DataSizing::FinalSysSizing(curSysNum).HeatSupHumRat;
-            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress,
-                                                                  "ReportCoilSelection::setCoilCoolingCapacity");
+            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
+                c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
             c->coilDesLvgEnth = Psychrometrics::PsyHFnTdbW(c->coilDesLvgTemp, c->coilDesLvgHumRat);
             if (DataAirSystems::PrimaryAirSystem(curSysNum).NumOAHeatCoils > 0) { // there is preHeating of the OA stream
                 c->oaPretreated = true;
             }
         }
 
     } else if (curZoneEqNum > 0 && allocated(DataSizing::FinalZoneSizing)) {
+        c->zoneNum.resize(1);
+        c->zoneName.resize(1);
+        if (allocated(DataZoneEquipment::ZoneEquipConfig)) c->zoneNum[0] = DataZoneEquipment::ZoneEquipConfig(curZoneEqNum).ActualZoneNum;
+        if (allocated(DataZoneEquipment::ZoneEquipConfig)) c->zoneName[0] = DataZoneEquipment::ZoneEquipConfig(curZoneEqNum).ZoneName;
```

Comment:
Second change added to allow zone name to be reported in Coil Sizing Details table.